pom.xml

```diff
@@ -253,9 +253,19 @@
 			<version>2.16</version>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
+			<artifactId>httpmime</artifactId>
+			<version>4.5.2</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.0</version>
+		</dependency>
```

Comment:
My mistake, they are not used in test classes. Lang3 is required in AutomationInterfaceBuilder.java and httpmime is used in WS.java